.changeset/config.json

```diff
@@ -6,6 +6,5 @@
   "linked": [],
   "access": "public",
   "baseBranch": "main",
-  "updateInternalDependencies": "patch",
-  "ignore": ["!@itwin/*"]
```

Comment:
Had to remove this because changesets was complaining.

```sh
$ pnpm changeset version

> itwinui-monorepo@0.0.0 changeset /Users/mayank/Desktop/Code/iTwinUI
> npx -y @changesets/cli "version"

🦋  error The package "@itwin/itwinui-css" depends on the ignored package "configs", but "@itwin/itwinui-css" is not being ignored. Please pass "@itwin/itwinui-css" to the `--ignore` flag.
🦋  error The package "@itwin/itwinui-react" depends on the ignored package "configs", but "@itwin/itwinui-react" is not being ignored. Please pass "@itwin/itwinui-react" to the `--ignore` flag.

ELIFECYCLE  Command failed with exit code 1.
```